src/lib/validationSchemas/specialistCreateSchema.js

```diff
@@ -0,0 +1,57 @@
+import { z } from 'zod';
+import { FormatOfWork, Gender } from '@prisma/client';
+import { PHONE_REGEX } from '@/lib/consts';
+
+const zString = fieldName => z
+  .string({
+    required_error: `${fieldName} is required`,
+    invalid_type_error: 'Must be a string',
+  })
+  .trim();
+
+const zStringWithMinMax = fieldName => zString(fieldName)
+  .min(2, {
+    message: 'Can not be empty',
```

Comment:
🟢 thats technically incorrect, as having 1 char is not empty, yet triggers error
maybe something like `Must have at least 2 chars` ?

Reply:
Makes sense